tasks/views.py

```diff
@@ -16,31 +16,38 @@ def view_tasks(request):
 
 
 def new_task(request):
-    if request.method == "POST":
-        form = TaskForm(request.user.id, request.POST)
-        taskForm = form.save(commit=False)
-        if form.is_valid():
-            try:
-                Task.create_task(taskForm.title, taskForm.created_by, taskForm.assignee, taskForm.priority,
-                                 taskForm.status, taskForm.description)
-
-                messages.success(request, 'Task was added successfully')
+    context = {'manager': False}
+    if not request.user.is_anonymous:
```

Comment:
```suggestion
    if request.user.is_authenticated():
```

If a user isn't anonymous he is authenticated. Also, I think that because I think you need to call it as a method as you are not in a template, which [Django calls for you](https://docs.djangoproject.com/en/dev/topics/templates/#accessing-method-calls)